src/components/map/ha-location-editor.ts

```diff
@@ -109,7 +109,7 @@ class LocationEditor extends LitElement {
 
     if (changedProps.has("hass")) {
       const oldHass = changedProps.get("hass") as HomeAssistant | undefined;
-      if (!oldHass || oldHass.themes?.darkMode === this.hass.themes?.darkMode) {
```

Comment:
Why is this no longer needed? We still init `themes` as `null`?

Reply:
Should not have been needed before the PR either. Basically no usages of `hass.themes` do check via `?` (e.g. none of the cards do).